packages/studio/src/App.tsx

```diff
@@ -3,8 +3,16 @@ import ActionsBar from "./components/ActionsBar";
 import Toast from "./components/Toast";
 import PreviewWithUseComponents from "./components/PreviewWithUseComponents";
 import LeftSidebar from "./components/LeftSidebar";
+import { useState } from "react";
+import { Tooltip } from "react-tooltip";
 
 export default function App() {
+  const [tooltipProps, setTooltipProps] = useState({
+    open: false,
+    position: { x: 0, y: 0 },
```

Comment:
Since we use position to determine the location of the tooltip, if we hover and cause a tooltip to appear then scroll, the component will move but the tooltip will not.  I think this issue is similar to your other highlighting scroll item!

Reply:
[Fixed](https://www.loom.com/share/3928db725dbb4613bc84c061c5f2655f)!

Reply:
just a quick follow-up-- I haven't tried it out myself, but is it possible to make the tooltip follow the component while you are hovered over it and scrolling instead of disappearing?  Not sure how complicated that would be, but wanted to check if you knew.

Reply:
It is possible I believe! I asked Tom and I believe he wanted it to just disappear on scroll. That seems to be pretty common behavior for react-tooltips.

Reply:
haha one last thing-- this might be a small nit, but if we have the disappearing behavior on the the scroll in the iFrame, should we have that same behavior when we scroll in the main window?  Right now, when scrolling in the main window, the tooltip follows the component.

Reply:
oooh kk, thanks!